_articles/dual-booting.md

```diff
@@ -27,6 +27,14 @@ If you wish to install Pop!\_OS here is a [link](/articles/install-pop/) to it.
 
 ## Install Windows 10
 
+### Create Install Media
+
+To create the Install Media on Pop!\_OS and Ubuntu the WoeUSB tool can be used and [here](https://www.dropbox.com/s/8emsfrqkqmf9km9/woeusb_3.3.1a_amd64.deb?dl=0) is a link to the latest build that works with 20.04 LTS. The Windows ISO file can be downloaded [here](https://www.microsoft.com/en-us/software-download/windows10ISO).
```

Comment:
To create the Install Media on Pop!\_OS and Ubuntu, use the [WoeUSB tool](https://www.dropbox.com/s/8emsfrqkqmf9km9/woeusb_3.3.1a_amd64.deb?dl=0). 

Once you have WoeUSB installed, download [The Windows ISO file] (https://www.microsoft.com/en-us/software-download/windows10ISO).